docs/en/observability/monitor-kubernetes.asciidoc

```diff
@@ -0,0 +1,223 @@
+[[monitor-kubernetes]]
+= Monitor Kubernetes: Observe the health and performance of your Kubernetes deployments
+
+*Open issues:* 
+
+* What assumptions can we make about the user wrt Kubernets knowledge? Can we
+assume that users are familiar with how Kubernetes works and the overall
+architecture?
+* Do we want to discuss prometheus monitoring?
+* Do we want to cover APM, Heartbeat, and Packetbeat in this tutorial? (current
+outline assumes that we do)
+
```

Comment:
Add: Do we want to provide some a sample app that users can deploy and build our examples around that?

Reply:
:+1: ,  that would be a good reference when covering users' questions/support etc